activiti-cloud-runtime-bundle-service/pom.xml

```diff
@@ -8,23 +8,42 @@
     <version>7.1.0-SNAPSHOT</version>
     <relativePath>../activiti-cloud-build</relativePath>
   </parent>
-  <groupId>org.activiti.cloud</groupId>
   <artifactId>activiti-cloud-runtime-bundle-service-parent</artifactId>
   <version>7.1.0-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>Activiti Cloud Runtime Bundle Service :: Parent</name>
   <modules>
-
     <module>activiti-cloud-runtime-bundle-dependencies</module>
     <module>activiti-cloud-services-runtime-bundle</module>
     <module>activiti-cloud-starter-runtime-bundle</module>
     <module>activiti-cloud-starter-runtime-bundle-it</module>
   </modules>
   <properties>
-    <activiti.version>7.1.341</activiti.version>
+    <activiti.version>0.0.1-dev-gicappa-GH-3688-preview-212-SNAPSHOT</activiti.version>
   </properties>
   <dependencyManagement>
     <dependencies>
+      <!-- 26/07/2021
+After upgrading spring-boot from 2.5.2 to 2.5.3, the
+build started failing because it was not anymore able
+to create the ApplicationContext.
+The error reported was:
+
+Caused by: org.springframework.context.ApplicationContextException: Failed to start bean 'inputBindingLifecycle'; nested exception is java.lang.NoSuchMethodError: org.springframework.integration.amqp.inbound.AmqpInboundChannelAdapter.<init>(Lorg/springframework/amqp/rabbit/listener/AbstractMessageListenerContainer;)V
+at org.springframework.context.support.DefaultLifecycleProcessor.doStart(DefaultLifecycleProcessor.java:181)
+
+The issue seems to be caused by this change in spring-integration-amqp:
+
+https://github.com/spring-projects/spring-integration/commit/76f77ccd2c5f94e000495357e66c4f99fffc6010
+
+Since the issue happens only due to the dependencies imported for testing it has been fixed rolling
+back the library spring-integration-amqp the the previous version (5.5.1)
+-->
+      <dependency>
+        <groupId>org.springframework.integration</groupId>
+        <artifactId>spring-integration-amqp</artifactId>
+        <version>5.5.1</version>
+      </dependency>
```

Comment:
You should be able to remove this, as it's already declared in `activiti-cloud-build-dependencies-parent`.

Reply:
My first implementation was relying on the definition declared in `activiti-cloud-build-dependencies-parent` but `activiti-cloud-runtime-bundle-service` was failing again. I tried to inspect on the possible dependency issue but the complexity of the dependencies where preventing me to do it in a proper time and was diverting me from the primary goal of the pull request.

Reply:
Fixed by the commit 559fbc1